src/itemset.js

```diff
@@ -191,6 +218,7 @@ define([
                     afterInsert: afterDynamicSelectInsert,
                     spec: {
                         itemsetData: itemsetDataSpec,
+                        filter: itemsetDataSpec,
```

Comment:
Previously itemset filters were not serialized for copy/paste, which means they were lost.